framework/test/jest/integration/utils/configs/config_cache.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const cacheConfig = () => ({
```

Comment:
Why don't we set the object to variable directly?

Reply:
This ensures that the cacheConfig is not overwritten by any of the tests.